### ISSUE TITLE
Plaid Link "Exit" button doesn't work after ITEM_NO_ERROR

### ISSUE THREAD

**Issue by dhoffman34** [plaid/link#322]
### A detailed description of the steps to reproduce the issue

1. Put an Item in the `ITEM_LOGIN_REQUIRED` state:  plaid.com/docs/#simulating-item_login_required-errors

2. Open an instance of Plaid Link in update mode for the Item

3. Open a second instance of Plaid Link in update mode for that Item

4. Authenticate in the first instance of Plaid Link using 'user_good' and 'pass_good' as your credentials

5. Try to authenticate in the second instance of Plaid Link using the same credentials

### What you expected to see?

A helpful message letting the user know their Item is already in a good state and they don't need to take any other action.

### What you saw instead?

An unhelpful message saying that an internal server error occured.

<details><summary>Screenshots that show the state of the UI (if applicable)</summary>

![Screen Shot 2019-05-06 at 2 45 02 PM](https://user-images.githubusercontent.com/4803968/57248566-a74ade00-7010-11e9-8fc2-b8da2b02d0d1.png)

</details>

**Comment by nathankot** [comment]
Thanks for reporting this @dhoffman34. So it sounds like while the first instance of Link behaved as expected, the second instance is failing with the bad error because the item is no longer in update mode by the time the second instance has completed the login flow.

I will file an internal issue for this - if you require further support with this or would like to escalate it please don't hesitate to file a support ticket through our dashboard!

**Comment by dhoffman34** [comment]
> So it sounds like while the first instance of Link behaved as expected, the second instance is failing with the bad error because the item is no longer in update mode by the time the second instance has completed the login flow

Exactly, yeah. I would have expected something like a helpful message to let the user know what to do in that case, rather than just an internal server error message.
